nomad/structs/structs.go

```diff
@@ -2757,21 +2757,15 @@ func (a *AllocatedTaskResources) Comparable() *ComparableResources {
 	return ret
 }
 
+// Subtract only subtracts CPU and Memory resources. Network utilization
+// is managed separately in NetworkIndex
```

Comment:
I think we'd need to update the `TestComparableResources_Subtract` test as it's currently computing the difference in [1]

[1] https://github.com/hashicorp/nomad/blob/1b787a15bf44f0c1ef19b8466e22bbe1616a6464/nomad/structs/structs_test.go#L1924-L1928